### PR TITLE
Update link to prettify plugin.

### LINKED ISSUE #7977: Prettify has incorrect link
The link to the Google-Code-Prettify links to the http://code.google.com/p/google-code-prettify/ which is currently archived and it should link to it's new home on GitHub https://github.com/google/code-prettify

### PR DESCRIPTION
**Pull request by RobJohnston**
Fixes #7977.

### CODE REVIEW

**Review by duboisp — APPROVED** [review]
I approve the changes, 

I noted that the [French short description](https://github.com/wet-boew/wet-boew/blob/master/site/pages/docs/ref/plugins-fr.hbs#L35) in the main plugin documentation page do not contains the link to the google prettify plugin. That means both description are currently not equivalent.

@RobJohnston It will be nice if you can fix the short description in French, but I don't think this PR should be blocked if you don't.

**Comment by RobJohnston** [comment]
I noticed that difference too. I can add it, but you'll have to give me the translation. ‎ Or should it be removed in the English?

Sent from my BlackBerry 10 smartphone on the Rogers network.
From: Pierre Dubois
Sent: Wednesday, June 7, 2017 16:01
To: wet-boew/wet-boew
Reply To: wet-boew/wet-boew
Cc: Rob Johnston; Mention
Subject: Re: [wet-boew/wet-boew] Update link to prettify plugin. (#8049)



@duboisp approved this pull request.

I approve the changes,

I noted that the French short description<https://github.com/wet-boew/wet-boew/blob/master/site/pages/docs/ref/plugins-fr.hbs#L35> in the main plugin documentation page do not contains the link to the google prettify plugin. That means both description are currently not equivalent.

@RobJohnston<https://github.com/robjohnston> It will be nice if you can fix the short description in French, but I don't think this PR should be blocked if you don't.

**Comment by duboisp** [comment]
Here the translation for [plugins-fr.hbs](https://github.com/wet-boew/wet-boew/blob/master/site/pages/docs/ref/plugins-fr.hbs#L35):

Replace: 
```html
<li><a href="prettify/prettify-fr.html">Prettify</a> - Affiche de la syntaxe des extraits de code source.</li>
```
By:
```html
<li><a href="prettify/prettify-fr.html">Prettify</a> - Coloration syntaxique des extraits de code source dans une page html en utilisant <a href="https://github.com/google/code-prettify" rel="external" hreflang="en">google-code-prettify</a>.</li>
```

**Comment by RobJohnston** [comment]
Translation from @duboisp added to PR.

File: site/pages/docs/ref/plugins-fr.hbs [hunk 1]
```diff
@@ -32,7 +32,7 @@
 	<li><a href="menu/menu-fr.html">Menu</a> - Une barre de menu interactive comprenant des sous-menus optionnels.</li>
 	<li><a href="texthighlight/texthighlight-fr.html">Mise en surbrillance de texte</a> - Met en surbrillance sur une page Web les mots choisis.</li>
 	<li><a href="footnotes/footnotes-fr.html">Notes de bas de page</a> - Une méthode cohérente et facile d’accès pour répertorier les notes de bas de page.</li>
-	<li><a href="prettify/prettify-fr.html">Prettify</a> - Affiche de la syntaxe des extraits de code source.</li>
+    <li><a href="prettify/prettify-fr.html">Prettify</a> - Coloration syntaxique des extraits de code source dans une page html en utilisant <a href="https://github.com/google/code-prettify" rel="external" hreflang="en">google-code-prettify</a>.</li>
```
Comment (duboisp) [inline site/pages/docs/ref/plugins-fr.hbs]: Can you change the 4 spaces indentation by a 1 tab space?

**Review by duboisp — CHANGES_REQUESTED** [review]

File: site/pages/docs/ref/plugins-fr.hbs (hunk 1 above)
Comment (RobJohnston) [inline site/pages/docs/ref/plugins-fr.hbs]: Done... damn Visual Studio!

Comment (nschonni) [inline site/pages/docs/ref/plugins-fr.hbs]: https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig will help you.

File: site/pages/docs/ref/plugins-fr.hbs [hunk 2]
```diff
@@ -1,4 +1,4 @@
----
+﻿---
```
Comment (duboisp) [inline site/pages/docs/ref/plugins-fr.hbs]: Having that diff line is weird,

**Review by duboisp — APPROVED** [review]

File: site/pages/docs/ref/plugins-fr.hbs (hunk 2 above)
Comment (nschonni) [inline site/pages/docs/ref/plugins-fr.hbs]: Might be worth checking in Notepad++ to check that the encoding is still UTF-8 (without BOM)

Comment (RobJohnston) [inline site/pages/docs/ref/plugins-fr.hbs]: I think that was it... stand by.

**Review by duboisp — APPROVED** [review]
Thanks @RobJohnston

**Pull request merged**